src/webui/api/torrentscontroller.cpp

```diff
@@ -835,3 +856,91 @@ void TorrentsController::removeCategoriesAction()
     for (const QString &category : categories)
         BitTorrent::Session::instance()->removeCategory(category);
 }
+
+QVariantList TorrentsController::getStickyTrackers(const BitTorrent::TorrentHandle* torrent)
+{
+    static const QString working("Working");
+    static const QString disabled("Disabled");
+    static const QString privateMsg("This torrent is private");
+
+    QVariantList stickyTrackers;
+
+    QVariantMap dht;
+    dht[KEY_TRACKER_URL] = "** [DHT] **";
```

Comment:
Not accustomed to list initialization? 
```c++
QVariantMap dht = {
    {KEY_TRACKER_URL, "** [DHT] **"},
    // ...
};
```

Reply:
I couldn't quite figure out the syntax for this. Thanks!